packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/telephony-line-phone-accessories.module.js

```diff
@@ -0,0 +1,15 @@
+import component from './telephony-line-phone-accessories.component';
+import routing from './telephony-line-phone-accessories.routing';
+import choice from './choice/telephony-line-phone-accessories-choice.component';
+import finalize from './finalize/telephony-line-phone-accessories-finalize.component';
+import resume from './resume/telephony-line-phone-accessories-resume.component';
+import shipping from './shipping/telephony-line-phone-accessories-shipping.component';
+
+angular
+  .module('managerApp')
+  .component('telephonyLinePhoneAccessories', component)
+  .component('telephonyLinePhoneAccessoriesChoice', choice)
```

Comment:
This could be splitted in smaller sub-modules, and each could declare a component

Reply:
It's already splitted into submodule which are add into the parent module.

Reply:
I think the comments is more about how to reuse theses components independently. Here, all these `telephonyLinePhoneAccessories` components are defined in the same files. So are they reusable independently? If yes, could it be declared in sub-modules?
Do you agree with this @marie-j ?

Reply:
OK I'll take a look to set them into sub-modules and re-test all.